homeassistant/components/velux/cover.py

```diff
@@ -73,17 +73,19 @@ def current_cover_position(self):
 
     @property
     def device_class(self):
-        """Define this cover as either window/blind/awning/shutter."""
-        if isinstance(self.node, Window):
-            return "window"
-        if isinstance(self.node, Blind):
-            return "blind"
-        if isinstance(self.node, RollerShutter):
-            return "shutter"
+        """Define this cover as either awning, blind, garage, gate, shutter or window."""
         if isinstance(self.node, Awning):
             return "awning"
+        if isinstance(self.node, Blind):
+            return "blind"
         if isinstance(self.node, GarageDoor):
             return "garage"
+        if isinstance(self.node, Gate):
+            return "gate"
+        if isinstance(self.node, RollerShutter):
+            return "shutter"
+        if isinstance(self.node, Window):
+            return "window"
```

Comment:
btw, do you have any idea, why the velux windows are shown as shutters in apple home.

(which has the effect, that all windows open, when you say "hey siri, open the shutters". :-) )

Reply:
I'm sorry, I don't know. I'm not quite there yet in my integration. I will definitely have a look at it, because I have Somfy roller shutters as well as Velux windows :-)